src/components/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from '../actions/index';
+import CategoryFilter from './CategoryFilter';
+
+const Navbar = ({ changeFilter }) => {
+  const handleFilterChange = e => {
+    changeFilter(e.target.value);
+  };
+  return (
+    <header className="bgWhite">
+      <nav className="navbar" id="myTopnav">
+        <a className="navbarBrand azure" href="/">
+          Book Store
+        </a>
+        <nav>
+          <a href="#home" className="navItem books tt_u">
+            Books
+          </a>
+          <CategoryFilter filterHandler={handleFilterChange} />
```

Comment:
Could you style this better and make the mouse recognise it as a clickable element? Currently it' too hidden and may not be obvious that it' changeable. Maybe add a `cursor: pointer;` CSS property, that way, hovering it makes it more obvious it's clickable. I would also recommend giving it a border when the mouse hovers it.